css/wishlist.css

```diff
@@ -0,0 +1,82 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap");
+
+* {
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
+  margin: 0;
+  padding: 0;
+  color: var(--secondary-dark);
+}
+
+body {
```

Comment:
Giving flex to body tag, can be messy for user, try giving a class and then apply styling to that.

Reply:
removed it!